CHANGELOG.md

```diff
@@ -22,6 +22,9 @@
 
 #### Enhancements
 
+* Adds `empty_string` opt-in rule.
```

Comment:
HI @iDevid! Take a look [here](https://github.com/realm/SwiftLint/blob/master/CONTRIBUTING.md#tracking-changes) for the format used on the changelog file. For instance, it must contain two trailing spaces on the last line describing the change.